application/src/main/java/org/togetherjava/tjbot/features/jshell/backend/JShellApi.java

```diff
@@ -19,21 +19,51 @@
 import java.net.http.HttpResponse.BodyHandler;
 import java.net.http.HttpResponse.BodyHandlers;
 
+/**
+ * Allows to interact with the unofficial JShell REST API of the Together-Java JShell backend
+ * project.
```

Comment:
might just drop a link to it as well. `<a href="...">JShell REST API</a>`

Reply:
I thought of that, but since the name of the API isn't fixed and might change, I am afraid that we forget to change this link.

Reply:
Changed, tell me what you think.